src/math/Color.js

```diff
@@ -546,6 +546,12 @@ class Color {
 
 	}
 
+	isBlack() {
```

Comment:
IMO, such a method is way too application specific for a `Color` class. We could now also add `isWhite()`, `isRed()` and so on...

Reply:
Hmm. My comment got lost, apparently. So here it is again...

> done!